frontend/components/ui/Table.tsx

```diff
@@ -36,20 +43,37 @@ export const TableWrapper = styled.div`
     }
     .ant-table-row:last-child {
       .ant-table-cell {
-        border-bottom: none;
+        border-bottom: ${({ $noBorder }) => ($noBorder ? 'none' : undefined)};
+
+        &:first-child {
+          border-bottom-left-radius: ${({ $noBorder }) =>
+            $noBorder ? undefined : '8px'};
```

Comment:
can we do this or 0px/none instead?
```suggestion
            $noBorder ? 'unset' : '8px'};
```

undefined is not a valid CSS value

Reply:
true but styled-components ignores `undefined` and `null` values — it doesn’t emit `border-bottom-left-radius: undefined;` — it simply skips that line. So the browser will just use the default (inherited or initial) style.